contrib/opentimelineio_contrib/adapters/ale.py

```diff
@@ -116,6 +117,33 @@ def _parse_data_line(line, columns, fps):
                 target_url=source
             )
 
+        # If available, collect cdl values in the same way we do for CMX EDL
+        cdl = {}
+
+        if 'CDL' in metadata:
+            cdl = _cdl_values_from_metadata(metadata['CDL'])
+            if cdl:
+                metadata.pop('CDL')
+
+        # If we have more specific metadata, let's use them
+        if 'ASC_SOP' in metadata:
+            cdl = _cdl_values_from_metadata(metadata['ASC_SOP'])
```

Comment:
Would you ever have both CDL and ASC_SOP? If so, this will clobber the earlier one.

Reply:
It can happen to have both CDL and ASC_SOP, so in this case I'd rather trust the ASC_SOP more than the generic CDL. Even though for now in all the samples I examined the values were pretty much the same.

Reply:
As long as it is documented that ASC_SOP takes precedence over CDL that should be fine.